server/routes/api/user.js

```diff
@@ -93,4 +93,34 @@ router.delete('/favorite/:id', async (req, res) => {
     res.json(rows);
 });
 
+router.get('/ramad-ads', async (req, res) => {
+    const userUpn = req.user
+
+    const { rows } = await db.query(`
+    SELECT json_build_object(
+        'id', ads.id,
+        'name', job_title,
+        'isClosed',is_close,
+        'role', json_build_object(
```

Comment:
`json_build_object`, `json_agg`, this is very cool!!!!! 💯 

Liked that such solution exists 👍 👍 👍